mapping.md

```diff
@@ -40,6 +40,8 @@ documentation when migrating._
 | `invert`            | `invert`            | `invertObj`         |
 | `isEmpty`           | `isEmpty`           | `isEmpty`           |
 | `isNil`             | `isNil`             | `isNil`             |
+| `isObjectType`      | `isObjectLike`      |                     |
```

Comment:
question: why would we opt for `isObjectType` rather than `isObjectLike` ?

Reply:
I would call it `isTypeOfObject` because I want names to be as close to their ECMAScript semantics, but I'm afraid that people would have difficulty finding it because they'd start typing `isObject...` and look for it via autocomplete.

I don't understand the semantics of "Like" in the context of `isObjectLike`, is a `RegExp` object-like? Is a `Class` with only `static` methods object-like? Because `typeof === "object"` is kinda the fallback in JS for anything that isn't something else, I feel that likeness to an object is not the key attribute being checked here...

Reply:
We need to add fuzzy search in the documentation site, and then add the mapping/aliases to the index so that people can search for different kinds of names coming from different other libraries and still find what they are looking for.